miraclelinux/meta-debian-extended
# heimdal: Update version from 7.5.0+dfsg-3 to 7.5.0+dfsg-3+deb10u1

# Purpose of pull request

This version fixes following CVEs.
- CVE-2019-14870
- CVE-2021-3671
- CVE-2021-44758
- CVE-2022-3437
- CVE-2022-41916
- CVE-2022-42898
- CVE-2022-44640

DLA: DLA 3206-1
# Test
## How to test

1. Build package (Add IMAGE_INSTALL_append += " heimdal heimdal-clients heimdal-servers heimdal-kdc heimdal-kcm heimdal-dev heimdal-multidev heimdal-doc" to local.conf)
2. Run commands
3. Check libraries are installed correctly.

## Test result

Run command with --version to check they are at least works.

```
root@qemuarm64:~# for cmd in afslog asn1_compile asn1_print gsstool heimtools hxtool kadmin.heimdal kdestroy.heimdal kf kgetcred kinit.heimdal klist.heimdal kpagsh kpasswd.heimdal krb5-config krb5-config.heimdal ksu.heimdal kswitch.heimdal ktutil.heimdal otp otpprint slc string2key verify_krb5_conf; do /usr/bin/$cmd --version 2>1 >/dev/null ; echo $?; done
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0


root@qemuarm64:~# for cmd in digest-service hprop hpropd iprop-log ipropd-master ipropd-slave kcm kdigest kimpersonate kstash ; do /usr/sbin/$cmd --version 2>1 >/dev/null ; echo $?; done
0
0
0
0
0
0
0
0
0
0
```
Installed libraries.

```
root@qemuarm64:~# for lib in /usr/lib/libgssapi.so.3.0.0 /usr/lib/libkrb5.so.26.0.0 /usr/lib/libheimbase.so.1.0.0 /usr/lib/heimdal/pkgconfig/krb5-gssapi.pc /usr/lib/heimdal/pkgconfig/krb5.pc /usr/lib/heimdal/pkgconfig/kadm-client.pc /usr/lib/heimdal/pkgconfig/kadm-server.pc /usr/lib/libasn1.so.8.0.0 /usr/lib/heimdal-servers/kadmind /usr/lib/heimdal-servers/kdc /usr/lib/heimdal-servers/kfd /usr/lib/heimdal-servers/kpasswdd /usr/lib/libsl.so.0.2.1 /usr/lib/libroken.so.18.1.0 /usr/lib/libotp.so.0.1.5 /usr/lib/libkadm5srv.so.8.0.1 /usr/lib/libkdc.so.2.0.0 /usr/lib/libkafs.so.0.5.1 /usr/lib/libhdb.so.9.2.0 /usr/lib/pkgconfig/heimdal-gssapi.pc /usr/lib/pkgconfig/heimdal-kadm-server.pc /usr/lib/pkgconfig/heimdal-kadm-client.pc /usr/lib/pkgconfig/krb5-gssapi.pc /usr/lib/pkgconfig/krb5.pc /usr/lib/pkgconfig/heimdal-krb5.pc /usr/lib/pkgconfig/kadm-client.pc /usr/lib/pkgconfig/kadm-server.pc /usr/lib/pkgconfig/kafs.pc /usr/lib/libkadm5clnt.so.7.0.1 /usr/lib/libheimedit.so.0.0.36 /usr/lib/libheimntlm.so.0.1.0 /usr/lib/libcom_err.so.1.1.3 /usr/lib/libhcrypto.so.4.1.0 /usr/lib/libhx509.so.5.0.0 /usr/lib/libheimsqlite.so.0.0.0 /usr/lib/libwind.so.0.0.0 ; do ls $lib ; done             
/usr/lib/libgssapi.so.3.0.0
/usr/lib/libkrb5.so.26.0.0
/usr/lib/libheimbase.so.1.0.0
/usr/lib/heimdal/pkgconfig/krb5-gssapi.pc
/usr/lib/heimdal/pkgconfig/krb5.pc
/usr/lib/heimdal/pkgconfig/kadm-client.pc
/usr/lib/heimdal/pkgconfig/kadm-server.pc
/usr/lib/libasn1.so.8.0.0
/usr/lib/heimdal-servers/kadmind
/usr/lib/heimdal-servers/kdc
/usr/lib/heimdal-servers/kfd
/usr/lib/heimdal-servers/kpasswdd
/usr/lib/libsl.so.0.2.1
/usr/lib/libroken.so.18.1.0
/usr/lib/libotp.so.0.1.5
/usr/lib/libkadm5srv.so.8.0.1
/usr/lib/libkdc.so.2.0.0
/usr/lib/libkafs.so.0.5.1
/usr/lib/libhdb.so.9.2.0
/usr/lib/pkgconfig/heimdal-gssapi.pc
/usr/lib/pkgconfig/heimdal-kadm-server.pc
/usr/lib/pkgconfig/heimdal-kadm-client.pc
/usr/lib/pkgconfig/krb5-gssapi.pc
/usr/lib/pkgconfig/krb5.pc
/usr/lib/pkgconfig/heimdal-krb5.pc
/usr/lib/pkgconfig/kadm-client.pc
/usr/lib/pkgconfig/kadm-server.pc
/usr/lib/pkgconfig/kafs.pc
/usr/lib/libkadm5clnt.so.7.0.1
/usr/lib/libheimedit.so.0.0.36
/usr/lib/libheimntlm.so.0.1.0
/usr/lib/libcom_err.so.1.1.3
/usr/lib/libhcrypto.so.4.1.0
/usr/lib/libhx509.so.5.0.0
/usr/lib/libheimsqlite.so.0.0.0
/usr/lib/libwind.so.0.0.0
```
